lib/ui/painting.dart

```diff
@@ -1534,9 +1535,10 @@ void decodeImageFromPixels(
   int width,
   int height,
   PixelFormat format,
+  int rowBytes,
```

Comment:
Please document what this argument does in the dartdocs.

This is a breaking change; please apply the breaking change polity (https://flutter.io/design-principles/#handling-breaking-changes). Alternatively, we could make this an optional trailing named argument with a default value.

Reply:
done - converted rowBytes to a named optional parameter